exporter/prometheusremotewriteexporter/factory.go

```diff
@@ -35,7 +43,11 @@ func createMetricsExporter(ctx context.Context, set exporter.CreateSettings,
 		return nil, errors.New("invalid configuration")
 	}
 
-	prwe, err := newPRWExporter(prwCfg, set)
+	retryOn429 := false
+	if metricsOriginFeatureGate.IsEnabled() {
+		retryOn429 = true
+	}
+	prwe, err := newPRWExporter(prwCfg, set, retryOn429)
```

Comment:
```suggestion
	prwe, err := newPRWExporter(prwCfg, set, metricsOriginFeatureGate.IsEnabled())
```

No need for an intermediate value here as the gate must be registered or there would have been a panic during runtime initialization.  That said, not sure that it makes sense to add this as an argument to this function at all.  Might be better just to set an unexported field on the `prwExporter` when initializing it.

Reply:
Fixed.